evalml/tests/automl_tests/test_automl.py

```diff
@@ -1851,31 +1851,58 @@ def __init__(self, parameters, random_state):
 
 @patch('evalml.pipelines.BinaryClassificationPipeline.score', return_value={"Log Loss Binary": 0.8})
 @patch('evalml.pipelines.BinaryClassificationPipeline.fit')
-def test_automl_error_callback(mock_fit, mock_score, X_y_binary, caplog):
+def test_automl_error_callback_none(mock_fit, mock_score, X_y_binary, caplog):
     X, y = X_y_binary
     msg = 'all your model are belong to us'
     mock_fit.side_effect = Exception(msg)
     automl = AutoMLSearch(X_train=X, y_train=y, problem_type="binary", error_callback=None, train_best_pipeline=False, n_jobs=1)
     automl.search()
     assert msg in caplog.text
 
+
+@patch('evalml.pipelines.BinaryClassificationPipeline.score', return_value={"Log Loss Binary": 0.8})
```

Comment:
How come splitting this test into many tests fixes the crashing worker problem?

Reply:
The dreaded windows test bug D:

Reply:
That I'm not sure.  We can't say conclusively that it does.  It fixed it "this time", but as @bchen1116 knows, sometimes just modifying something seems to affect the presence of the crashing worker.  We were discussing maybe having a nightly testing of main to see if main, outside of tests via merging, exhibits this crash going forward.  But really I guess this will be a question of whether we accept the additional lines due to splitting the test in the hopes of fixing the bug going forward, or not.